src/frontends/paddle/src/op/set_value.cpp

```diff
@@ -36,28 +26,77 @@ std::shared_ptr<Node> handle_maximum_index(Output<Node>& node, const Output<Node
     return std::make_shared<default_opset::Select>(mask, update_node, node);
 }
 
-bool is_contain_minus(const std::vector<int64_t> vec) {
-    for (int64_t i : vec) {
-        if (i < 0)
-            return true;
+void normalize(std::vector<int64_t>& vec, const Output<Node> intput, const std::vector<int64_t> axes_vec) {
+    for (size_t i = 0; i < axes_vec.size(); i++) {
+        if (vec[i] < 0) {
+            auto x_dim = std::stoll(intput.get_partial_shape()[axes_vec[i]].to_string());
```

Comment:
Can we simply use 
`auto x_dim = intput.get_partial_shape()[axes_vec[i]];`
here?